hack/make/generate-index-listing

```diff
@@ -41,7 +41,7 @@ create_index() {
 	IFS=$'\n';
 
 	# pretty sweet, will mimick the normal apache output
-	for L in $(find -L . -mount -depth -maxdepth 1 -type f ! -name 'index' -printf "<a href=\"%f\">%-44f@_@%Td-%Tb-%TY %Tk:%TM  @%f@\n"|sort|sed 's,\([\ ]\+\)@_@,</a>\1,g');
+	for L in $(find -L . -mount -depth -maxdepth 1 -type f ! -name 'index' -printf "<a href=\"%f\">%f|@_@%Td-%Tb-%TY %Tk:%TM  @%f@\n"|sort|column -t -s '|' | sed 's,\([\ ]\+\)@_@,</a>\1,g');
```

Comment:
This is actually incorrect, I had to change it to `-printf "<a href=\"X_X\">%f|@_@%Td-%Tb-%TY %Tk:%TM  @%f@\n"|column -t -s'|'|sed 's,X_X">\([^\ ]\+\)\([\ ]\+\)@_@,\1">\1</a>\2,g'|sort);`